apps/graphql/src/types/output/Itinerary.js

```diff
@@ -1,21 +1,48 @@
 // @flow
 
-import { GraphQLObjectType, GraphQLInt, GraphQLString } from 'graphql';
+import {
+  GraphQLObjectType,
+  GraphQLInt,
+  GraphQLString,
+  GraphQLList,
+} from 'graphql';
 import GlobalID from '@kiwicom/graphql-global-id';
+import { GraphQLDateTime } from 'graphql-iso-date';
 
 export type Itinerary = {|
   +id: string,
 |};
 
+const routeType = new GraphQLObjectType({
+  name: 'Route',
+  fields: {
+    airline: { type: GraphQLString },
+    cityFrom: { type: GraphQLString },
+    cityTo: { type: GraphQLString },
+    flyFrom: { type: GraphQLString },
+    flyTo: { type: GraphQLString },
+    id: { type: GraphQLString },
+    localArrival: { type: GraphQLDateTime },
+    utcArrival: { type: GraphQLDateTime },
+    localDeparture: {
+      type: GraphQLDateTime,
+    },
+    utcDeparture: { type: GraphQLDateTime },
```

Comment:
There is quite a lot of from and to going on here. Maybe we should add
Arrival and Departure object. 

```
name: 'Route',
  fields: {
	arrival: {type: SomeReusableType },
	departure: {type: SomeRuesableType }
  }

name: 'SomeRuesableType', 
fields: {
	city: {type: string },
	localTime: {type: GraphqlDate},
	utcTime: {type: GraphqlDate},
}
```

Do you see the point?

Reply:
That's good idea, thanks!

Reply:
Done